presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java

```diff
@@ -32,5 +57,77 @@ public void testDelete()
         // Hive connector currently does not support row-by-row delete
     }
 
+    @Test
+    public void testBroadcastSpatialJoinContains()
```

Comment:
why you added these tests to hive connector? Why not to have these tests in geospatial module?

Reply:
I'll move these tests to presto-geospatial.